aten/src/ATen/templates/NativeFunctions.cpp

```diff
@@ -0,0 +1,10 @@
+// ${generated_comment}
```

Comment:
Hmmm.. maybe rename to `DispatchKeyNativeFunctions.cpp`, like the existing `DispatchKeyNativeFunctions.h` template?